addons/account/models/account_reconcile_model.py

```diff
@@ -153,6 +200,15 @@ class AccountReconcileModel(models.Model):
         default='old_first',
         tracking=True,
     )
+    counterpart_type = fields.Selection(
+        selection=[
+            ('general', 'Journal Entry'),
```

Comment:
Maybe I missed something but `Journal Entry` doesn't create any entry... just do the regular journal items as before. The wording is a bit confusing to me

Reply:
I think it comes from the fact that the type of rule is "Button to generate counterpart entry", but if @chklop agrees we can change the counterpart type to Journal Item instead.

Reply:
@smetl Journal Entry is better, compared to the other options (Customer Invoices and Vendor Bills). I know that technically that option is not about an entry, but it will be clearer for the user. Remind yourself that unmatched Bank transactions should not be entries, in the mind of the accountant they're created when you validate the matching, not before. The whole trick of "posted entries but on a suspense account" on we use in Odoo is not obvious or easy to understand for accountants, so "Journal Entry" makes more sense.